addons/sys_core/initSettings.sqf

```diff
@@ -127,3 +127,24 @@
     true,
     {[_this] call EFUNC(api,setRevealToAI)}
 ] call CBA_Settings_fnc_init;
+
+// Notification Settings
```

Comment:
Should we pursue this method, or should we use the notification system on the upper left corner and change the active color to green when intercom/racks are active?